modules/compose/compose_api.go

```diff
@@ -267,12 +267,10 @@ func (d *dockerCompose) Down(ctx context.Context, opts ...StackDownOption) error
 	return d.composeService.Down(ctx, d.name, options.DownOptions)
 }
 
-func (d *dockerCompose) Up(ctx context.Context, opts ...StackUpOption) error {
+func (d *dockerCompose) Up(ctx context.Context, opts ...StackUpOption) (err error) {
```

Comment:
Is there a way to avoid naked returns?

Reply:
> Is there a way to avoid naked returns?

If we really want to avoid that, we have to check `err != nil` or not in every single `return` statement, and if `err != nil` cleanup by sending a `termSignal`, this is more error prune in my opinion, what do you think?

Reply:
I refactor the code to avoid naked return, let me know how you like it 😄